src/Models/HydrostaticFreeSurfaceModels/mg_implicit_free_surface_solver.jl

```diff
@@ -56,17 +60,31 @@ step `Δt`, gravitational acceleration `g`, and free surface at the `n`-th time-
 """
 function MGImplicitFreeSurfaceSolver(grid::AbstractGrid, 
                                      settings = nothing,
-                                     gravitational_acceleration = nothing, 
+                                     gravitational_acceleration = g_Earth,
+                                     reduced_dim = (false, false, false),
```

Comment:
@elise-palethorpe are you sure about this? shouldn't it be `(false, false, true)`?